pkgs/development/python-modules/pysmartthings/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, aiohttp
+}:
+
+buildPythonPackage rec {
+  pname = "pysmartthings";
+  version = "0.7.6";
+
+  src = fetchPypi {
```

Comment:
Switch to the source from GitHub. Check `test-requirements.txt` for the needed inputs.

```bash
Check whether the following modules can be imported: pysmartthings
setuptoolsCheckPhase
Executing setuptoolsCheckPhase
running test
WARNING: Testing via this command is deprecated and will be removed in a future version. Users looking for a generic test entry point independent of test runner are encouraged to use tox.
WARNING: The directory '/homeless-shelter/.cache/pip' or its parent directory is not owned or is not writable by the current user. The cache has been disabled. Check the permissions and owner of that directory. If executing pip with sudo, you may want sudo's -H flag.
WARNING: Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x7ffff61dd520>: Failed to establish a new connection: [Errno -2] Name or service not known')': /simple/tox/
WARNING: Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x7ffff61dd370>: Failed to establish a new connection: [Errno -2] Name or service not known')': /simple/tox/
WARNING: Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x7ffff61dd4f0>: Failed to establish a new connection: [Errno -2] Name or service not known')': /simple/tox/
WARNING: Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x7ffff61ddc10>: Failed to establish a new connection: [Errno -2] Name or service not known')': /simple/tox/
WARNING: Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x7ffff61dda60>: Failed to establish a new connection: [Errno -2] Name or service not known')': /simple/tox/
ERROR: Could not find a version that satisfies the requirement tox<4.0.0,>=3.5.0
ERROR: No matching distribution found for tox<4.0.0,>=3.5.0
[...]
```